app/views/smart_answers/custom_result.html.erb

```diff
@@ -7,6 +7,28 @@
   <meta name="robots" content="noindex">
 <% end %>
 
+<% content_for :phase_banner do %>
+  <% feedback_message = capture do %>
+    We’re still building this part of GOV.UK –
+    <%= link_to "give us your feedback (opens in a new tab)",
+      "/done/next-steps-for-your-business",
+      class: "govuk-link",
+      data: {
+        module: "gem-track-click",
+        "track-action": "/done/next-steps-for-your-business",
+        "track-category": "Internal Links Clicked",
+        "track-label": "give us your feedback"
+      }
+    %>
+    to help us improve it.
+  <% end %>
+
+  <%= render "govuk_publishing_components/components/phase_banner", {
+    phase: "beta",
```

Comment:
This attribute will be removed when the `phase_banner` will be able to render without a phase.